search/include/pcl/search/impl/brute_force.hpp

```diff
@@ -69,16 +69,16 @@ pcl::search::BruteForce<PointT>::nearestKSearch (
 //////////////////////////////////////////////////////////////////////////////////////////////
 template <typename PointT> int
 pcl::search::BruteForce<PointT>::denseKSearch (
-    const PointT &point, int k, std::vector<int> &k_indices, std::vector<float> &k_distances) const
+    const PointT &point, int k, Indices &k_indices, std::vector<float> &k_distances) const
 {
   // container for first k elements -> O(1) for insertion, since order not required here
   std::vector<Entry> result;
   result.reserve (k);
   std::priority_queue<Entry> queue;
   if (indices_)
   {
-    std::vector<int>::const_iterator iIt =indices_->begin ();
-    std::vector<int>::const_iterator iEnd = indices_->begin () + std::min (static_cast<unsigned> (k), static_cast<unsigned> (indices_->size ()));
+    auto iIt =indices_->begin ();
```

Comment:
`cbegin` and `cend` if you're using `auto` (elsewhere too)

Reply:
Done